scripts/auto_tick.sh

```diff
@@ -29,21 +29,36 @@ set -e
 source ~/mc/etc/profile.d/conda.sh
 conda env list
 conda activate botenv
+# clone graph repo to get cached graph
+git clone --depth 1 https://github.com/nsls-ii-forge/auto-tick-graph.git
+cp names.txt ./auto-tick-graph/
+cd ./auto-tick-graph/
 # list of packages to be updated (in graph)
 echo "=================="
 cat names.txt
 echo "=================="
-# clean current directory of bot files
-auto-tick clean -y
 # create graph with node_attrs/* and graph.json
 time graph-utils make -o nsls-ii-forge -c -f names.txt -m 10
```

Comment:
Will this procedure be faster with the graph from the clonned repo?

Reply:
Making the graph should be faster since it will only be updating nodes it has seen before and adding any new nodes it hasn't seen before. I'm not sure how much of a speedup this is because it still should be checking for updated recipe files.